website/src/pages/index.module.css

```
@@ -1,339 +1,326 @@
 .home {
+  --home-gutter: clamp(1rem, 4vw, 4rem);
   overflow: hidden;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)